.github/CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 <h3>New features since last release</h3>
 
+* Added the templates `ArbitraryUnitary` and `ArbitraryStatePreparation` that use
+  `PauliRot` gates to perform an arbitrary unitary and prepare an arbitrary basis
+  state with the minimal number of parameters.
+  [(#590)](https://github.com/XanaduAI/pennylane/pull/590)
+
+  An example usage of the templates would be to define a trainable quantum kernel:
+
+  ```python
+    dev = qml.device('default.qubit', wires=3)
+
+    @qml.template
+    def kernel_embedding(angles1, angles2, x):
+      qml.templates.ArbitraryStatePreparation(angles1, wires=[0, 1, 2])
+      qml.templates.IQPEmbedding(features=x, wires=[0, 1, 2])
+      qml.templates.ArbitraryUnitary(angles2, wires=[0, 1, 2])
+      qml.templates.IQPEmbedding(features=x, wires=[0, 1, 2])
+
+    @qml.qnode(dev)
+    def circuit(angles1, angles2, x, y):
+        kernel_embedding(angles1, angles2, x)
+        qml.inv(kernel_embedding(angles1, angles2, y))
+
+        return qml.probs([0, 1, 2])
+  ```
```

Comment:
What about a simpler example for this purpose?

```suggestion
  The templates could be used as follows:

  ```python
    dev = qml.device('default.qubit', wires=3)

    @qml.device(dev)
    def circuit(angles1, angles2, x):
          qml.templates.ArbitraryStatePreparation(angles1, wires=[0, 1, 2])
          qml.templates.IQPEmbedding(features=x, wires=[0, 1, 2])
          qml.templates.ArbitraryUnitary(angles2, wires=[0, 1, 2])
          return qml.expval(qml.PauliZ(wires=0))
```